flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -593,6 +593,40 @@ public enum SchedulerType {
                                             code(SchedulerType.AdaptiveBatch.name()))
                                     .build());
 
+    @Documentation.Section({
+        Documentation.Sections.EXPERT_SCHEDULING,
+        Documentation.Sections.ALL_JOB_MANAGER
+    })
+    public static final ConfigOption<Boolean> SPECULATIVE_ENABLED =
```

Comment:
I perfer to add a `ADAPTIVE_BATCH_SCHEDULER` prefix, the same goes for the other two options.

Reply:
I prefer to keep the option name not too long, as long as the we can identify it is part of speculative execution.
I will rename `BLOCK_SLOW_NODE_DURATION` to `SPECULATIVE_BLOCK_SLOW_NODE_DURATION`.